src/command_modules/azure-cli-batchai/azure/cli/command_modules/batchai/_params.py

```diff
@@ -0,0 +1,199 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+from enum import Enum
+
+from azure.cli.command_modules.vm._actions import get_vm_sizes
+from azure.cli.core.commands.parameters import ignore_type, location_type, resource_group_name_type, enum_choice_list, \
+    get_one_of_subscription_locations
+from azure.cli.core.sdk.util import ParametersContext
+from azure.mgmt.storage.v2017_06_01.models import SkuName
```

Comment:
Does __BatchAI__ take a hard dependency on the __2017_06_01__ storage API version? Otherwise, you can get the `SkuName` using [`azure.cli.core.profiles.get_sdk`](https://github.com/Azure/azure-cli/blob/effdd3e0e2bb7946b93b3fe3ab06ee14674dd2c9/src/azure-cli-core/azure/cli/core/profiles/__init__.py#L50).

The benefit of this method is that it will get the correct type base on the current API profile user is using.

Reply:
No, i do not care about the version -) 
from azure.mgmt.storage.models import SkuName